README.md

```diff
@@ -132,12 +132,42 @@ currently, there are verification tasks for the ILP32 (32-bit) and the LP64 (64-
 
 In order to obtain verification tasks from the programs and specifications in the repository,
 a simple task-definition mechanism is used.
-For each program, the repository contains a .yml file that specifies 
-  - the subject files (the parts that a program consists of),
-  - the results (for each specification of the program, the expected outcome is indicated), and
-  - the parameters to be used for verification.
+For each program, the repository contains a .yml file that specifies the following items:
+  - `format_version`: the version of the format (a version string)
+  - `input_files`: the subject program files or directories
+    (a list of file or directory names that a program consists of)
+  - `properties`: the properties that constitute the specification of the program,
+    each consisting of the following items:
+    - `property_file`: file that contains a property definition
+    - `expected_verdict`: the intended verification result (`true` or `false`)
+    - `subproperty`: a subproperty of the property that is violated
+      in cases where the property is a conjunction of subproperties (optionally for verdict `false`) 
```

Comment:
> optionally for verdict `false`

This sounds like it is mandatory for verdict `true`.

Proposal 1:
```
- `subproperty` (optional): a subproperty of the property that is violated
      in cases where the property is a conjunction of subproperties (for verdict `false`)
```

Proposal 2:
```
- `subproperty`: a subproperty of the property that is violated
      in cases where the property is a conjunction of subproperties (optional and only for verdict `false`)
```

Proposal 1 allows the key for verdict `true`, Proposal 2 forbids it.

Reply:
Used Proposal 1. Thanks. Not sure if we need the restriction to true.